src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,36 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int SIZE = 12;
+    public static final int EMPTY = -1;
+    public static final int MAX_CAPACITY = 11;
+    private int[] numbers = new int[SIZE];
 
-    public int total = -1;
+    public int total = EMPTY;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == EMPTY;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == MAX_CAPACITY;
```

Comment:
Wartość MAX_CAPACITY 11 powinna być zależna od wartości SIZE, czyli powinno to być na zasadzie SIZE-1